content/blog/2018/istio-authorization/index.md

```diff
@@ -0,0 +1,211 @@
+
+* **Roles allow grouping of attributes.** Roles are groups of permissions, which specifies the actions you are allowed
+to perform on a system. Users are grouped based on the roles within an organization. You can define the roles and reuse
+them for different cases.
+
+* **It is easier to understand and reason about who has access.** The RBAC concepts map naturally to business concepts.
+For example, a DB admin may have all access to DB backend services, while a web client may only be able to view the
+frontend service.
+
+* **It reduces unintentional errors.** RBAC policies make otherwise complex security changes easier. You won't have
+duplicate configurations in multiple places and later forget to update some of them when you need to make changes.
+
+On the other hand, Istio's authorization system is not a traditional RBAC system. It also allows users to define **conditions** using
+[combinations of attributes](/docs/reference/config/authorization/constraints-and-properties/). This gives Istio
+flexibility to express complex access control policies. In fact, **the “RBAC + conditions” model
+that Istio authorization adopts, has all the benefits an RBAC system has, and supports the level of flexibility that
+normally an ABAC system provides.** You'll see some [examples](#examples) below.
+
+### High performance
+
+Because of its simple semantics, Istio authorization is enforced on Envoy as a native authorization support. At runtime, the
+authorization decision is completely done locally inside an Envoy filter, without dependency to any external module.
+This allows Istio authorization to achieve high performance and availability.
+
+### Work with/without primary identities
+
+Like any other RBAC system, Istio authorization is identity aware. In Istio authorization policy, there is a primary
+identity called `user`, which represents the principal of the client.
+
+In addition to the primary identity, you can also specify any conditions that define the identities. For example,
+you can specify the client identity as “user Alice calling from Bookstore frontend service”, in which case,
+you have a combined identity of the calling service (`Bookstore frontend`) and the end user (`Alice`).
+
+To achieve best security, it is strongly recommended that users enable [authentication features](/docs/concepts/security/#authentication),
+and use authenticated identities in authorization policies. However, strongly authenticated identity is not required
+for using authorization. Istio authorization works with or without identities. If you are working with a legacy system,
+you may not have mutual TLS or JWT authentication setup for your mesh. In this case, the only way to identify the client is, say,
+through IP. You can still use Istio authorization to control which IP addresses or IP ranges are allowed to access your service.
+
+## Examples
+
+In [authorization task page](/docs/tasks/security/role-based-access-control/), you have already learned how to
```

Comment:
The [authorization task](/docs/tasks/security/role-based-access-control/) shows you how to use